core/apps/ame-e2e/src/integration/editor/edit-property.cy.ts

```diff
@@ -520,16 +520,9 @@ describe('Test edit property', () => {
         .then(() => cyHelp.clickSaveButton())
         .then(() => cy.getUpdatedRDF())
         .then(rdf => {
-          expect(rdf).to.contain(':property1 a samm:Property;\n' + '    samm:characteristic :Characteristic1.');
-          expect(rdf).to.contain(
-            ':AspectDefault a samm:Aspect;\n' +
-              '    samm:properties ([\n' +
-              '  samm:property :property1;\n' +
-              '  samm:optional true\n' +
-              ']);\n' +
-              '    samm:operations ();\n' +
-              '    samm:events ().\n',
-          );
+          expect(rdf).to.contain(':property1 a samm:Property;');
+          expect(rdf).to.contain('samm:characteristic :Characteristic1.');
+          expect(rdf).to.contain('samm:optional true');
```

Comment:
Here we dont know if the AspectDefault contains their properties and their all the other elements.